pkg/sdk/poc/generator/plain_struct.go

```diff
@@ -53,6 +53,10 @@ func (v *plainStruct) OptionalNumber(dbName string) *plainStruct {
 	return v.Field(dbName, "*int")
 }
 
+func (v *plainStruct) PlainStructField(name string, plainStruct *plainStruct) *plainStruct {
```

Comment:
It's not used anywhere (or I missed something), do you think it's needed?

Reply:
I used it earlier, forgot to remove.